src/lightning_app/core/work.py

```diff
@@ -537,6 +537,14 @@ def _cleanup_calls(calls: Dict[str, Any]):
                         final_statuses.append(status)
                 calls[call_hash]["statuses"] = final_statuses
 
+    def start(self):
+        """Starts LightingWork component via L.CloudCompute."""
+        if self.status.stage == WorkStageStatus.STOPPED:
```

Comment:
shouldn't we include anything by `NOT_STARTED`?

Reply:
I see it now, the others are handled by the implicit caching semantics

Reply:
This would break if we do so, as user won't be protecting their `.start` method. Therefore, we need to rely on the cache mechanism of the run method.